src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/PpudCreateOrUpdateSentenceRequest.kt

```diff
@@ -6,12 +6,12 @@ data class PpudCreateOrUpdateSentenceRequest(
   val custodyType: String,
   val dateOfSentence: LocalDate,
   val licenceExpiryDate: LocalDate?,
-  val mappaLevel: String,
+  val mappaLevel: String?,
   val releaseDate: LocalDate?,
   val sentenceLength: SentenceLength?,
-  val espCustodialPeriod: PpudYearMonth? = null,
-  val espExtendedPeriod: PpudYearMonth? = null,
-  val sentenceExpiryDate: LocalDate? = null,
+  val espCustodialPeriod: PpudYearMonth? = null, // never set in UI - should we remove it and all associated code?
+  val espExtendedPeriod: PpudYearMonth? = null, // never set in UI - should we remove it and all associated code?
```

Comment:
Can I presume there are other fields that exist within PPUD that we do no concern ourselves with setting like these two.
I am wondering if this object represents the complete scope of creating a sentence or if it is already the partial set that pertains to CaR. If it's the later, then I would definitely move to remove these are that would make them demonstrably obsolete.